### ISSUE TITLE
Confusion around meaning of name in Records and UI

### ISSUE THREAD

**Issue by humphd** [DevelopingSpace/starchart#373]
A recent change has broken our code for string the `name` of a record in the db and UI.

In the example below, you can see an old record I created, which shows the full name (ignore the missing `.` which is #359).  The second record shows only the subdomain part, and is missing the rest:

<img width="1077" alt="Screenshot 2023-03-17 at 3 04 20 PM" src="https://user-images.githubusercontent.com/427398/225995543-8ec8c8fe-5def-46e7-9583-c0d065665d8c.png">

On staging, in the database, a similar thing is happening:

<img width="1595" alt="Screenshot 2023-03-17 at 3 06 56 PM" src="https://user-images.githubusercontent.com/427398/225996206-ae6ac820-2302-4651-9db9-75a660208acb.png">

Is this related to @Genne23v's recent work to separate `subdomain` and `fqdn`?  What should we be storing for `name` in the Records table (cc @cychu42).

**Comment by humphd** [comment]
I think the bug is in `add-record-flow.server.ts`:

```ts
export const addDnsRequest = async (data: AddDnsRequestData) => {
  const { username, type, subdomain, value } = data;

  const fqdn = buildDomain(username, subdomain);

  // Before running workflow. Add a record to DB
  const recordId = await createUserRecord({
    username,
    type,
    name: subdomain,           // <--- should this be `fqdn` instead?
    value,
  });
```

**Comment by humphd** [comment]
Or is the issue that the front-end needs to construct the `fqdn` from this `subdomain` + the user's `baseDomain`?

**Comment by Eakam1007** [comment]
If we are renaming the name column to `subdomain`, maybe this should be done by front-end when displaying the records?

**Comment by humphd** [comment]
Yes, we can fix in front-end

**Comment by Myrfion** [comment]
@humphd As far as I understand this issue got resolved in #369, right?

**Comment by humphd** [comment]
Yes, we're good.
